components/os/db/events.go

```diff
@@ -7,28 +7,38 @@ import (
 	"time"
 
 	"github.com/leptonai/gpud/pkg/sqlite"
+
 	_ "github.com/mattn/go-sqlite3"
 )
 
 const (
-	TableNameMachineBootIDs = "machine_boot_ids"
+	DeprecatedTableNameMachineBootIDs = "machine_boot_ids"
+	TableNameBootEvents               = "component_os_boot_events"
 
-	ColumnBootID          = "boot_id"
-	ColumnBootUnixSeconds = "boot_unix_seconds"
+	ColumnBootID    = "boot_id"
+	ColumnTimestamp = "timestamp"
 )
 
-func CreateTableBootIDs(ctx context.Context, db *sql.DB) error {
+func CreateTable(ctx context.Context, db *sql.DB) error {
+	if _, err := db.ExecContext(ctx, fmt.Sprintf(`DROP TABLE IF EXISTS %s`, DeprecatedTableNameMachineBootIDs)); err != nil {
```

Comment:
do we delete the table very time we restart gpud?

Reply:
Yes, but will be removed in the next/following GPUd release (e.g., v0.4.1)

Reply:
is it for migration? what if a user upgraded directly from 0.3.9 to 0.4.1? will it mess things up?

Reply:
i think we should simply use a different table name if we want to significantly change the schema

Reply:
> from 0.3.9 to 0.4.1? will it mess things up?

Should be safe. We always create table (if not exists). This creation statement will succeed in either case.

Worst case, older versions will have dangling, unused tables. We can keep this drop statement for a long time.

Reply:
so we are okay to lose all data during gpud restarts?

Reply:
@ccding Yes.